### ISSUE TITLE
Improve resize behavior

### ISSUE THREAD

**Issue by petebankhead** [qupath/log-viewer#56]
...if possible (I know it's awkward with JavaFX & I often can't figure out how).

Specifically, I can decrease the window size - but at some point items are just cut.

Also, there is no padding for the log symbols on the left column, so that they look very cramped.

Screenshot showing both of these things:

<img width="574" alt="Log viewer resizing" src="https://github.com/qupath/log-viewer/assets/4690904/9993e34e-32d4-4f5a-98c4-30edb1484ab2">

**Comment by Rylern** [comment]
What is the expected behavior here?

**Comment by Rylern** [comment]
The window (or `Stage` in JavaFX) is not part of the LogViewer. The LogViewer only provides a `BorderPane`, but the window containing the `BorderPane` can have a smaller size than the `BorderPane` (see [here](https://stackoverflow.com/a/12688604)).

So I think it's the client (QuPath here) that can set a minimum size to the window.

**Comment by petebankhead** [comment]
You're right, thanks!

**Comment by petebankhead** [comment]
I've set the minimum size here: https://github.com/qupath/qupath/pull/1333

That works, but I had to hard-code some values. If I requested the minimum size from the log viewer itself (i.e. the `BorderPane`), then it did not return a useful values.

Is there any way to query the appropriate minimum size from the `LogViewer`, so that hard-coded values aren't required - and it would adapt to any future changes in the `LogViewer` implementation?

If not, the current approach is fine for now.
